src/character.h

```diff
@@ -178,6 +178,19 @@ struct social_modifiers {
         return *this;
     }
 };
+
+struct consumption_event {
+    time_point time;
+    itype_id type_id;
+    uint64_t component_hash = 0;
+
+    consumption_event( const item &food ) {
```

Comment:
Use member initialisation list:
```suggestion
    consumption_event( const item &food ) : time( calendar::turn ) {
```

Reply:
Good call.